tiatoolbox/wsicore/wsireader.py

```diff
@@ -3167,36 +3168,69 @@ def time(string: str) -> datetime:
             "raw": raw,
         }
 
+    def _get_ome_xml(self) -> ElementTree.Element:
+        """Parse OME-XML from the description of the first IFD (page).
+
+        Returns:
+            ElementTree.Element:
+                OME-XML root element.
+
+        """
+        description = self.tiff.pages[0].description
+        return ElementTree.fromstring(description)
+
     def _parse_ome_metadata(self) -> dict:
+        """Extract OME specific metadata.
+
+        Returns:
+            dict:
+                Dictionary of kwargs for WSIMeta.
+
+        """
         # The OME-XML should be in each IFD but is optional. It must be
         # present in the first IFD. We simply get the description from
         # the first IFD.
-        from defusedxml.ElementTree import fromstring as et_from_string
-
         description = self.tiff.pages[0].description
```

Comment:
you are using this line twice once here and then in `_get_ome_xml`. you can get this from `_get_ome_xml`

Reply:
This line is being removed. It should only be used on one line in `_get_ome_xml`.

Reply:
how about this 
https://github.com/TissueImageAnalytics/tiatoolbox/blob/6d714f4d2dcf5bed053d0f08f0792176b2a5cd60/tiatoolbox/wsicore/wsireader.py#L3193